src/main/java/me/iblitzkriegi/vixio/expressions/channel/builder/ExprLastChannelBuilder.java

```diff
@@ -11,6 +11,7 @@
 import org.bukkit.event.Event;
 
 public class ExprLastChannelBuilder extends SimpleExpression<ChannelBuilder> {
+
     static {
         Vixio.getInstance().registerExpression(ExprLastChannelBuilder.class, ChannelBuilder.class, ExpressionType.SIMPLE,
```

Comment:
Missing docs

Reply:
this isnt my expression

Reply:
Can you add a //TODO and I'll come back to it?

Reply:
i believe you were looking at an old commit again, there are docs here